internal/users/cache/testdata/multiple_users_and_groups_with_brokers.db.yaml

```diff
@@ -32,6 +32,6 @@ UserToGroups:
   "3333": '{"UID":3333,"GIDs":[33333,99999]}'
```

Comment:
what’s now the difference between "multiple_users_and_groups.db.yaml` and that file? I think there is none, no? So either we remove one test, or we reverse the test to have one without `UserToBroker` (which I don’t think it’s relevant)

ditto with `internal/users/testdata/db/multiple_users_and_groups_with_brokers.db.yaml`

Reply:
I don't think we should remove tests, just remove the redundant file and replace the file used in the tests, no?

Reply:
but those 2 tests will have the exact same definition no?

Reply:
Actually... It is possible for the user to be in the database and have no broker. We set the default broker for the user in `AcctMgmt`, but the user is added to the cache in `Authenticate`. So, in theory, `Authenticate` could pass but, if something fails in `AcctMgmt`, the user never gets a default broker...

I'll update the fixture file to have a user (probably `user4`) without a default broker. Wdyt?

Reply:
good point! Yeah, that’s the best, and so, we convert this special subtest to a "without broker" instead of "with broker" (which is the default)

Reply:
I was thinking about using the same file, but keeping a user without a broker. I'm not opposed to creating another fixture for those specific cases, though (just maybe it's not necessary)

Reply:
> ditto with internal/users/testdata/db/multiple_users_and_groups_with_brokers.db.yaml

It seems this file was not removed, was it? I don’t have a checkout but as I don’t see any reference to it, it should now be a dangling symlink or is it GitHub not displaying this properly?

Reply:
Yeah, that's precisely it :smiling_face_with_tear: